src/server.tsx

```diff
@@ -75,17 +75,23 @@ const buildEpic = async (
         highlighted,
     };
 
-    // Only log for now - as Frontend does this and we may have bugs
-    // We'll need to do more than log when we start our DCR test though
+    // Don't render the Epic if our targeting checks fail
     if (shouldNotRenderEpic(targeting)) {
-        console.log(`Should not render for targeting data: ${JSON.stringify(targeting)}`);
+        if (process.env.LOG_TARGETING === 'true') {
+            console.log(`Renders Epic false for targeting: ${JSON.stringify(targeting)}`);
+        }
```

Comment:
I wonder if extracting the env var condition and `console.log` out to a function would be worth doing? Only because this creates a little bit of visual noise which most of the time, when reading this code, you probably aren't so interested in. Slightly inspired by [Tell Don't Ask](https://thoughtbot.com/blog/tell-dont-ask) What do you reckon? E.g.

```js
const logTargeting = (message :string) => {
  if (process.env.LOG_TARGETING === 'true') {
    console.log(msg);
  }
}

// Later on...
logTargeting(`Renders Epic false for targeting: ${JSON.stringify(targeting)}`);
```

Something like that?
```

Reply:
Yeah totally worth doing, good shout